### PR TITLE
Migrate from php-ml to rubix/ml

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

File: lib/Service/Classification/ImportanceClassifier.php [hunk 1]
```diff
@@ -350,19 +365,10 @@
 		 * Ref https://en.wikipedia.org/wiki/F1_score
 		 */
 		$this->logger->debug("classification report: " . json_encode([
-			'recall' => $report->getRecall(),
-			'precision' => $report->getPrecision(),
-			'f1Score' => $report->getF1score(),
+			'recall' => $recallImportant,
```
Comment (nickvergessen) [inline lib/Service/Classification/ImportanceClassifier.php:368]: Is this correct? You use the same var twice?

Comment (nickvergessen) [inline lib/Service/Classification/ImportanceClassifier.php:368]: $precisionImportant?

Comment (ChristophWurst) [reply lib/Service/Classification/ImportanceClassifier.php:368]: right. fixed :)

**Comment by ChristophWurst** [comment]
CI fails because of https://github.com/nextcloud/mail/pull/3160

**Pull request merged**